src/main/resources/init_db.sql

```diff
@@ -56,3 +56,7 @@ CREATE TABLE `cars_drivers`  (
 ) ENGINE = InnoDB CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Dynamic;
 
 SET FOREIGN_KEY_CHECKS = 1;
+
+alter table drivers
+    add login varchar(50) null,
+    add password varchar(11) null;
```

Comment:
better add these columns to CREATE TABLE `drivers` script 
Also, you have to validate these parameters (login varchar(50), password varchar(11)) on the API side because you haven't the same constraints in your ValidationService and vice versa you validate that login and password  can't be null but don’t have the same constraints  here
this can cause unexpected exceptions to occur